awx/ui_next/src/util/validators.jsx

```diff
@@ -76,6 +77,16 @@ export function integer(i18n) {
   };
 }
 
+export function number(i18n) {
+  return value => {
+    const str = String(value);
+    if (/^[0-9]*(\.[0-9]*)?$/.test(str)) {
```

Comment:
Probably should also handle:
  - big numbers `/^[0-9]*(\.[0-9]*)?$/.test(String(999999999999999999999.9))`
  - negative numbers `/^[0-9]*(\.[0-9]*)?$/.test(String(-2.2))`